.github/workflows/stable_diffusion_1_5_cpp.yml

```diff
@@ -69,6 +62,13 @@ jobs:
           source openvino_sd_cpp/bin/activate
           optimum-cli export openvino --model botp/stable-diffusion-v1-5 --task stable-diffusion models/stable_diffusion_v1_5_ov/FP16
 
+      - name: Build app
+        working-directory: ${{ env.WORKING_DIRECTORY }}
+        run: |
+          source ${{ env.OV_INSTALL_DIR }}/setupvars.sh
+          cmake -DCMAKE_BUILD_TYPE=Release -S ./ -B ./build/
+          cmake --build ./build/ --config Release --parallel
```

Comment:
can we have app build at the top?
The idea is to ensure that app can be built and only then, spend 10mins to install dependencies and download / convert model

Reply:
Fixed